cloud/terraform/templates.go

```diff
@@ -305,14 +306,25 @@ echo $PRIVATEIP > /tmp/.ip
 	"PUBLICIP=`ip route get 8.8.8.8 | awk '{printf \"%s\", $NF; exit}'`" + `
 
 
-kubeadm init --apiserver-bind-port ${PORT} --token ${TOKEN} --kubernetes-version v${CONTROL_PLANE_VERSION} \
-             --apiserver-advertise-address ${PUBLICIP} --apiserver-cert-extra-sans ${PUBLICIP} ${PRIVATEIP} \
-             --service-cidr ${SERVICE_CIDR}
+KUBEADM_FLAGS=" --apiserver-bind-port ${PORT} --token ${TOKEN} --kubernetes-version v${CONTROL_PLANE_VERSION}"
```

Comment:
Please break this out so we are adding one flag per line.

So something like:

KUBEADM_FLAGS="--foo"
KUBEADM_FLAGS=KUBEADM_FLAGS+"--bar"

This will make reading this section easier, and adding/removing flags safer.